sdks/java/core/src/main/java/org/apache/beam/sdk/io/PubsubIO.java

```diff
@@ -955,11 +961,14 @@ public void populateDisplayData(DisplayData.Builder builder) {
         super.populateDisplayData(builder);
 
         builder
-            .addIfNotNull(DisplayData.item("timestampLabel", timestampLabel))
-            .addIfNotNull(DisplayData.item("idLabel", idLabel));
+            .addIfNotNull(DisplayData.item("timestampLabel", timestampLabel)
+              .withLabel("Timestamp Label Attribute"))
+            .addIfNotNull(DisplayData.item("idLabel", idLabel)
```

Comment:
It might make sense to add some string constants for the keys/labels used in common places. Eg., "idLabel". It will make it easier to ensure that if we change the name of that item we change it everywhere.